AdminAPI.md

```diff
@@ -88,3 +88,24 @@ When a webhook is dispatched, the record `topic` is appended as a path component
  * `verified`: (string) whether the presentation is verified: `true` or `false`
  * `auto_present`: (boolean) prover choice to auto-present proof as verifier requests
  * `error_msg`: the previous error message
+
+## API Standard Behaviour
+
+The best way to develop a new admin API or protocol is to follow one of the existing protocols, such as the Credential Exchange or Presentation Exchange.
+
+The `routes.py` file contains the API definitions - API endpoints and payload schemas (note that these are not the Aries message schemas).
+
+The payload schemas are defined using [marshmallow](https://marshmallow.readthedocs.io/) and will be validated automatically when the API is executed (using a middleware).  (This raises a status `400` HTTP response with an error message if the schema validation fails.)
```

Comment:
422 - unless 400 includes the whole 400-series

Reply:
Noted thanks, I'll update the doc